src/components/dataset/DatasetAccess.jsx

```diff
@@ -20,7 +31,7 @@ function DatasetAccess(props) {
       dispatch(removeDatasetPolicyMember(dataset.id, removableEmail, role));
     };
   };
-  const { horizontal, policies, userRoles } = props;
+  const { horizontal, policies, userRoles, showHelp, classes } = props;
   const stewards = getRoleMembersFromPolicies(policies, DatasetRoles.STEWARD);
   const custodians = getRoleMembersFromPolicies(policies, DatasetRoles.CUSTODIAN);
   const snapshotCreators = getRoleMembersFromPolicies(policies, DatasetRoles.SNAPSHOT_CREATOR);
```

Comment:
The ticket did not ask for snapshot creators to get their block, but they did have a column. I decided to leave these in there instead of removing them completely. Is that okay?

Reply:
yeah, I think that makes sense